src/CMakeLists.txt

```diff
@@ -21,6 +21,7 @@ set(SOURCES
 set(HEADERS
   ../include/mc_rbdyn_urdf/api.h
   ../include/mc_rbdyn_urdf/urdf.h
+  ../include/mc_rbdyn_urdf/kukaiiwaURDF.h
```

Comment:
Shouldn't be there.

Reply:
It has been removed and included in my project.